skills/lark-meeting/scripts/meeting_qa_bootstrap.py

```
@@ -0,0 +1,645 @@
+    has_more = bool(block.get("has_more"))
+    page_token = block.get("page_token", "") or ""
+    return items, has_more, page_token
+
+
+def fetch_calendar_agenda(today: str) -> tuple[Optional[list[dict[str, Any]]], Optional[str]]:
+    """
+    获取今日日程中未来时间的会议（calendar +agenda），覆盖 vc +search 无法搜到的未来会议。
+    vc +search 只能搜到已结束且用户参与过的会议，calendar +agenda 可补全未来会议场景。
+    仅返回 start_time 在当前时间之后的日程（未来会议），已过期的日程由 vc +search 覆盖。
+    返回 (events, scope_hint)；scope_hint 不为空表示缺少权限，调用方应提示用户授权。
+    字段实测：data.events[] 每项含 event_id / summary / start_time / end_time / status 等。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)